erpnext/patches/v12_0/update_end_date_and_status_in_email.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import unicode_literals
+import frappe
+from frappe.utils import add_days, getdate, today
+
+def execute():
+    email_campaign = frappe.get_all('Email Campaign')
```

Comment:
Add a check here to fix the failing Patch Test:

```
if frappe.db.exists('DocType', 'Email Campaign'):
        email_campaign = frappe.get_all('Email Campaign')
```
And then the rest of the code inside the if block